velox/expression/fuzzer/ExpressionFuzzer.cpp

```diff
@@ -361,7 +361,8 @@ bool isDeterministic(
     return typeAndMetadata->second.deterministic;
   }
 
-  // functionName must be a special form.
+  // functionName must be a special form or a function whose determinism cannot
```

Comment:
> a function whose determinism cannot be established.

What would be an example of such a function?

Reply:
This happens on those functions need argument generators. `typeFuzzer.fuzzArgumentTypes` returns false, so the input `argTypes` of isDeterministic function is empty.